metis/lib/server/controllers/upload_controller.rb

```diff
@@ -71,7 +80,19 @@ def upload_start
       metis_uid: metis_uid,
     ).first
 
-    raise Etna::BadRequest, 'No matching upload' unless upload
+    if !upload
+      require_params(:project_name, :bucket_name, :file_path)
```

Comment:
Note that you don't have to require_params for things that are already in the route, as they are guaranteed to be present. This is maybe a more future-proof way to write the controller, though.